scripts/west_commands/sign.py

```diff
@@ -575,7 +575,10 @@ def sign(self, command, build_dir, build_conf, formats):
         if '-k' not in sign_config_extra_args + args.tool_args:
             # rimage requires a key argument even when it does not sign
             cmake_default_key = cache.get('RIMAGE_SIGN_KEY', 'key placeholder from sign.py')
-            extra_ri_args += [ '-k', str(sof_src_dir / 'keys' / cmake_default_key) ]
+            if os.path.exists(cmake_default_key):
```

Comment:
```suggestion
            if pathlib.Path(cmake_default_key).exists:
```

I've been told `pathlib` is the future.